SampleSize/SampleSizeCalculation.Rmd

```diff
@@ -0,0 +1,261 @@
+---
+title: "Sample size calculation"
+author: "Author"
+date: "21 4 2022"
+output:
+  github_document: default
+bibliography: references.bib
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, 
+                      warning = FALSE)
+```
+
+
+# Background
+
+In clinical research, one of the first technical hurdles to a project is the sample size calculation - how many participants would you need to include to be able to answer your research question? They are required by funding bodies, ethics and safety regulators. They can also be important to the sponsor - without a good sample size calculation, how can you tell if you will be able to achieve your aims?
+
+<!-- Figure here of clinician/statistician discussion?? Michael Coslovsky used to have a funny one... -->
+
+Speaking with a statistician about sample size calculation can be frustrating for both parties. The investigator just wants a number that they can write in their proposal/protocol/grant application. The statistician cannot give a number without various details. Hence, a good understanding from both sides is primordial.
+The aim of the study has to be well defined and all the needed assumptions educated guessed.
+
+There are two important concepts to understand before we move forward: Type 1 error and Type 2 error. 
+
+#### Type 1 error 
+
+A type 1 error is the rejection of the null hypothesis when it is actually true (i.e. a false positive - e.g. a pregnant man). This is controlled by the selection of $\alpha$, the test significance level, which is normally set at 0.05 (which represents incorrectly rejecting the null hypothesis 5% of the time). For more exploratory analyses, it might be relaxed to 0.1. 
+
+#### Type 2 error
+
+Type 2 errors are failing to reject the null hypotheses when they are actually false (i.e. a false negative - e.g. telling a pregnant woman that she is not pregnant). This is controlled by the power, which is generally set at 0.8. 
+
+
+### Two main paradigms:
+
+#### Null hypothesis significance testing (NHST) 
+
+The aim is to assess if there is a statistical difference between e.g. two groups. For example
+
+- does medication X lower Y more than medication Z
+- does surgery X results in fewer Y than surgery Z
+
+#### Precision based approaches
+The focus is rather on estimating a quantity to a given accuracy. For example, we might expect that 40% of people have a given event within a certain time frame, but we want to estimate that 40% with a specific precision, in other words within a certain interval, e.g. 5% (2.5% on either side). We are interested in the width of the confidence interval (CI). It is also possible to estimate width of the CI for differences (e.g. mean differences) and relative effects (e.g. odds ratios, relative risks).
+
+### What the statistician needs to know
```

Comment:
I would add "(at least)", because this covers only the simplest SSE

Reply:
you're referring to line 48 I suppose?

Reply:
yes (e.g., you are briefly referring to non-inferiority trials, for which the statistician would also need to know the non-inferiority margin)